views/docs.ejs

```diff
@@ -30,12 +30,12 @@
 
```

Comment:
Yes, it is. The original `/docs` path to the documentation is still available but is now being used to load the docs as an iframe as described in the pull request (#133) description. This had to be done to facilitate the docs' tab being shown as active as angular will not detect a path outside the app.